src/app/home/home.component.html

```diff
@@ -1,21 +1,81 @@
-<div fxLayoutAlign="center center">
+
+
+<div class = "container" 
+ fxLayout="column" fxLayoutAlign="center center"
+ fxLayout.sm="row" 
+ fxLayout.xs="row" 
+ fxLayoutAlign.xs="center center" 
+ fxLayoutAlign.sm="center center"
+ fxLayoutAlign.gt-sm="center center"
+ fxFlexAlign="center center"
+ style="margin-top:10em">
+
+ <h1> {{title}}</h1>
   <form >
-    
-      <input type="text" #restaurantbox="ngForm" style="width: 350px; height:30px" placeholder="hungry? what do you wanna eat" aria-label="Number" mdInput [formControl]="restaurant" [mdAutocomplete]="restau">
-      <input type="text" #locationbox="ngForm" style="width: 350px; height:30px" placeholder="where?" aria-label="Number" mdInput [formControl]="location" [mdAutocomplete]="loc">
+      <input type="text" #restaurantbox="ngForm"  name="rest"  style="width: 25em; height:2.5em" placeholder="hungry? what do you wanna eat" aria-label="Number" matInput [formControl]="restaurant" [matAutocomplete]="restau">
+      <input type="text" #locationbox="ngForm"  name="area" style="width: 25em; height:2.5em" placeholder="where?" aria-label="Number" matInput [formControl]="location" [matAutocomplete]="loc">
```

Comment:
Why have we used aria-label as "Number"? 
Shouldn't it be "Restaurant Name" for line 15 and "Location Name" for line 16?